torch/csrc/jit/passes/utils/memory_dag.cpp

```diff
@@ -100,13 +110,8 @@ void MemoryDAG::collectAllContainedMemoryLocationsImpl(
 }
 
 bool MemoryDAG::mayContainAliasImpl(const Element* a, const Element* b) const {
-  MemoryLocations all_a_mlocs;
```

Comment:
is this just a formatting change ? (for my own understanding)

Reply:
no.`getAllContainedMemoryLocations()` is more efficient than `collectAllContainedMemoryLocations(x)` because it can return a const reference to the cached value if it exists rather than copying the cached value into x.